Validation/MajorRoads/UBA-DE_HVS_to_END_DF7_10_Roads.halex

```diff
@@ -136,6 +136,26 @@
         <setting name="separator">,</setting>
         <setting name="escape">\</setting>
     </resource>
+    <resource action-id="eu.esdihumboldt.hale.io.instance.read.source" provider-id="eu.esdihumboldt.hale.io.xls.reader.instance">
```

Comment:
Now the sample data are added to the commit accidentally

Reply:
Oops, fixed